src/cascadia/TerminalApp/SnippetsPaneContent.xaml

```diff
@@ -225,11 +225,22 @@
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
 
-        <TextBlock x:Name="_title"
-                   x:Uid="SnippetPaneTitle"
-                   Grid.Row="0"
-                   Margin="4"
-                   FontSize="24" />
+        <Grid Grid.Row="0"
+              Margin="4"
+              HorizontalAlignment="Stretch">
+            <TextBlock x:Name="_title"
+                       x:Uid="SnippetPaneTitle"
+                       HorizontalAlignment="Left"
+                       FontSize="24" />
+            <Button x:Uid="CloseSnippetsPaneButton" HorizontalAlignment="Right" Click="_closePaneClick">

```

Comment:
```suggestion
            <Button x:Uid="CloseSnippetsPaneButton" HorizontalAlignment="Right" Click="Close">
```

does this work?

Reply:
Nope, cause `Close()` doesn't take the two arguments needed for a `Click` handler

Reply:
I'd thought the XAML compiler was flexible and used metadata to determine the arity of the event handler, but it appears I had mixed it up with something else.